FirebaseMessaging/Tests/UnitTests/FIRMessagingTokenOperationsTest.m

```diff
@@ -438,4 +442,11 @@ - (void)stubInstallations {
   OCMStub([_mockInstallations authTokenWithCompletion:authTokenWithCompletionArg]);
 }
 
+- (void)stubHeartbeatInfo {
+  _mockHeartbeatInfo = OCMClassMock([FIRHeartbeatInfo class]);
+  NSInteger sdkAndGlobalHeartbeatRequestedCode = 3;
```

Comment:
We can define it as a test property or a const to share between the method and the tests using it.

Reply:
While applying this feedback, I realized I could just use `FIRHeartbeatInfoCodeCombined` because the `FIRHeartbeatInfoCode` enum namespace is available in this file. Is this preferred over a test property/const?

Reply:
Using existing constant SGTM

Reply:
SG, thanks for all the feedback. I'll merge on green.